src/service.rs

```diff
@@ -1503,6 +1503,17 @@ where
     /// Poll listen connections
     #[inline]
     fn listen_poll(&mut self) {
+        if !self
```

Comment:
How the system restores and this condition becomes `false`?

Reply:
some session shutdown it will become `true`